os/net/lwip/src/include/lwip/sockets.h

```diff
@@ -475,6 +476,13 @@ typedef struct ip_mreq {
 #define SELWAIT_T u8_t
 #endif
 
+/* use in poll testcases */
+struct poll_test_args {
```

Comment:
How about moving this structure definition into `os_api_test_drv.h`?
Because it is only used for testcase.